extension/src/plots/data/index.ts

```diff
@@ -59,22 +59,19 @@ export class PlotsData extends BaseData<{
       ...args
     )
 
+    this.notifyChanged({ data, revs })
```

Comment:
[F] We don't want to delay updating the rest of the app whilst we mess around looking for files to watch.